src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(/[;:]/)
```

Comment:
[CODE KNOWLEDGE] - You are using `split` method with a regular expression that matches both `;` and `:`. This causes the property names and values to be mixed in the same array, which can lead to incorrect results if the property value contains `:`. It would be better to split the source string by `;` first to get the style declarations, and then split each declaration by `:` to get the property name and value.